### PR TITLE
ARXIVNG-1081: error message wording and format

### PR DESCRIPTION
**Pull request by eawoods**
This is the beginnings of the page suggested by @erickpeirson to link and further explain error messages to users.
It only contains UI errors at present and not log-level errors.

Let me know if I missed any! Not expecting this to be complete in its current form but hopefully a good start.

Ignore the sass file change, that was an artifact of a different branch that I just realized ended up propagating ahead of the next feature.

### CODE REVIEW

File: submit/templates/submit/error_messages.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{% extends "submit/base.html" %}
+
+{% block within_content %}
+<h2 class="is-size-4">Errors and Help</h2>
+
+<h3>File Upload Errors</h3>
+  <dl>
+    <dt>File name contains unsupported characters. Renamed <code>file_name</code> to <code>new_file_name</code>. Check and modify any file paths that reference this file.</dt>
```
Comment (erickpeirson) [inline submit/templates/submit/error_messages.html]: Any way to do direct links to these definitions?

Comment (eawoods) [reply submit/templates/submit/error_messages.html]: Yes! I was undecided on what nomenclature we might want for them - meant to ask about that.
Do we have a list of errors with shortnames or programmatic names that make sense to use as # links? Any other suggestions for a schema for naming these? Some of the titles get long, might make difficult (if readable) link names.

Comment (eawoods) [reply submit/templates/submit/error_messages.html]: Sounds good. I will boldly make up a set of initial slug/code names, then! New commit coming soon.

**Comment by eawoods** [comment]
OK....calling this done as a first pass.

I added the route /error for INTERNAL use, to make it easier to read the list and figure out whether the wording was more or less consistent. Some editing remains (see the multiple unsupported-characters entries, and unwritten failed-doc error), but in light of recent conversations about messaging schema this should probably be called done.

Here's a screenshot of the page - not beautiful, but functional at least.

screen shot 2018-09-05 at 3 54 06 pm

**Comment by erickpeirson** [comment]
This looks excellent; merge when ready!

**Pull request merged**